R/calculate.R

```diff
@@ -134,14 +137,14 @@ calculate_exposure_by_technology_and_company <- function(asset_type,
       setdiff(exposure_by_technology_and_company$company_name,
               exposure_by_technology_and_company_filtered$company_name)
       )
-    cat(
-      "      >> When filtering out holdings with exposures of 0 value, dropped rows for",
-      n_companies_pre - n_companies_post, "out of", n_companies_pre, "companies\n"
-    )
-    cat("        >> percent loss:", percent_loss, "\n")
-    cat("        >> affected companies:\n")
+    paste_write(
+      format_intend_1(), "When filtering out holdings with exposures of 0 value, dropped rows for",
```

Comment:
are we talking about the two different types of indents here? if so: spelling. If not: intent? still spelling.
Or something else?

Reply:
I starred at intend for quite a while, knew sth was off but did not get what, fixing to indent